packages/orm/objection/src/utils/createRelationshipMapping.ts

```
@@ -1,21 +1,29 @@
-  return (targetClass: any) => ({
-    [propertyKey]: {
-      relation,
-      modelClass: opts?.type || Metadata.getType(target, propertyKey),
-      join: createJoinKeys(targetClass, target, String(propertyKey), relation, opts),
-      modify: opts?.modify,
-      filter: opts?.filter,
-      beforeInsert: opts?.beforeInsert
-    }
-  });
+  return (targetClass: any) => {
+    opts = opts
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)